packages/plugins/src/plugins/morpho/lendingFetcher.ts

```diff
@@ -0,0 +1,129 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { getAddress } from 'viem';
+import { Fetcher, FetcherExecutor } from '../../Fetcher';
+import {
+  morphoMarketsCachePrefix,
+  networkIdToMorphoContract,
+  platformId,
+} from './constants';
+import { getEvmClient } from '../../utils/clients';
+import { ElementRegistry } from '../../utils/elementbuilder/ElementRegistry';
+import { Cache } from '../../Cache';
+import { MorphoMarketRes } from './types';
+import { morphoContractABI } from './utils/abis';
+
+export function getLendingFetcher(networkId: EvmNetworkIdType): Fetcher {
+  const executor: FetcherExecutor = async (owner: string, cache: Cache) => {
+    const markets = await cache.getItem<MorphoMarketRes['markets']['items']>(
+      morphoMarketsCachePrefix,
+      {
+        networkId,
+        prefix: morphoMarketsCachePrefix,
+      }
+    );
+    if (!markets || markets.length === 0) {
+      return [];
+    }
+
+    const client = getEvmClient(networkId);
+    const multicallResponse = await client.multicall({
+      contracts: markets.flatMap((market) => [
+        {
+          address: getAddress(networkIdToMorphoContract[networkId]),
+          functionName: 'position',
+          abi: morphoContractABI,
+          args: [market.uniqueKey, owner],
+        },
+        {
+          address: getAddress(networkIdToMorphoContract[networkId]),
+          functionName: 'market',
+          abi: morphoContractABI,
+          args: [market.uniqueKey],
+        },
```

Comment:
```suggestion
        } as const,
        {
          address: getAddress(networkIdToMorphoContract[networkId]),
          functionName: 'market',
          abi: morphoContractABI,
          args: [market.uniqueKey],
        } as const,
```

IINM, adding ` as const` here will allow us to remove the forced-casting below ?

If `as const` isn't enough, please see https://github.com/Octav-Labs/sonarwatch-portfolio/pull/42#discussion_r2031622624 from @BitWonka 's PR. This should allow you to remove the forced cast below ( `as [bigint, bigint, bigint]`).